<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MunchkinDialog</class>
 <widget class="QWidget" name="MunchkinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QPushButton" name="btn_ClearLog">
     <property name="text">
      <string>Очистить лог</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn_Send">
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chckBox_SendOnEnterPress">
     <property name="text">
      <string>Отправлять по нажатию &quot;Enter&quot;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTextBrowser" name="txtBrowser_TextLog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="txtEdit_UserText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
